Added Weather-JS as a project (#137)
Codesee-io/oss-port: projects/DeepNinja07x/Weather-JS.mdx

---
name: Weather-JS.
repoUrl: https://github.com/DeepNinja07x/Weather-JS

> description: It's a simple API based weather web application made using HTML5, CSS3, and JavaScript.

Reviewer: ```suggestion
description: A simple API-based weather web application made using HTML5, CSS3, and JavaScript.
```

languages: 
  - JavaScript
  - HTML / CSS 
tags: # A list of keywords relevant to the contributors for this project
  - First Timer Friendly
  - Environment
  - Tools
avatar: x-icon.png
websiteUrl: https://deepninja07x.github.io/Weather-JS/
twitterUrl: https://twitter.com/Deepraj_Rakshit
currentlySeeking: # (optional) These fields are searchable and very helpful for contributors to match well with your project
  - Developers
  - Backend Developers
  - Frontend Developers
  - DevOps
  - Designers
  - Maintainers
  - Code Reviewers
  - Mentors
  - Testers
featuredMap: # (optional) A CodeSee Map to serve as a visual starting point for contributors
  url: https://app.codesee.io/maps/3c7f8c50-21b2-11ec-82ba-4f649219975d
  description: A breakdown of the project structure of the Weather-JS app.
contributionOverview: # (optional) Provide additional context to help contributors match well with your project
  mainLocation: India
  idealEffort: 2 PRs a month
  isMentorshipAvailable: false
---

<!-- Divide your content into sections using our handy shortcodes. Each section is optional -->

<Overview>

> It's a simple API based weather web application made using HTML5, CSS3, and JavaScript. It's working process is simple. It takes the name of the City as input from the user and create a HTTP request and send it to Openweathermap.org API to get back the deatils in the form of a JSON Response. Out of all the details the following details are shortlisted:

Reviewer: ```suggestion
A simple API-based weather web application made using HTML5, CSS3, and JavaScript. Its working process is simple: it takes the name of the City as input from the user and creates an HTTP request and sends it to Openweathermap.org API to get back the details in the form of a JSON Response. Out of all the details the following details are shortlisted:
```

* Temperature of the City
* Climatic condition of the City
* Humidity
* Wind Speed (Km/h)

</Overview>

<Contributing>

> The current feature requests have been raised in the Issues tab of my repository. Kindly check them out. I am also open to new feature suggestion from you guys, so please raise them in the issue tab!

Reviewer: ```suggestion
The current feature requests have been raised in the Issues tab of the repository. Kindly check them out. I am also open to new feature suggestions from all, so please raise them in the issue tab!
```
